src/akkudoktoreos/class_strompreis.py

```diff
@@ -22,51 +24,56 @@ def repeat_to_shape(array, target_shape):
 
 
 class HourlyElectricityPriceForecast:
-    def __init__(self, source, cache_dir="cache", charges=0.000228, prediction_hours=24):  # 228
-        self.cache_dir = cache_dir
-        os.makedirs(self.cache_dir, exist_ok=True)
-        self.cache_time_file = os.path.join(self.cache_dir, "cache_timestamp.txt")
-        self.prices = self.load_data(source)
+    def __init__(self, source: str | Path, config: AppConfig, charges=0.000228):  # 228
+        self._cache_dir = config.working_dir / config.directories.cache
```

Comment:
For permanent installation it would be nice to have free choice over the directory structure. Support for absolute paths would be great (maybe build the full path directly in AppConfig)

Reply:
for this PR I just wanted to replace the current functionality. And would it really be needed to have different paths? They are all set from cwd or the provided user path.

Reply:
Fair point for a different PR. I'm thinking of Linux directory structure that splits up a package across the file system.